### PR TITLE
Add new FatalFunc method to logger to override behavior for Fatal-level logging

### PR DESCRIPTION
**Pull request by autarch**

### CODE REVIEW

**Comment by huan-Mongo** [comment]
@autarch I removed myself from the reviewer since I am out till next Thursday. If it still needs review then, you can feel free to assign me as reviewer. Thanks!

**Review by edobranov — APPROVED** [review]
LGTM!

File: log_test.go [hunk 1]
```diff
@@ -1039,3 +1039,22 @@ func TestUnmarshalTextLevel(t *testing.T) {
 		})
 	}
 }
+
+func TestFatalFunc(t *testing.T) {
```
Comment (huan-Mongo) [inline log_test.go:1043]: Could you also create a test case to make sure the fatal function can be passed down to the child logger created by a `With()` function?

**Review by huan-Mongo — CHANGES_REQUESTED** [review]
Looks good! Just have a question about the test case.

File: log_test.go (hunk 1 above)
Comment (autarch) [reply log_test.go:1043]: Added.

**Review by huan-Mongo — APPROVED** [review]
LGTM!

**Pull request merged**